src/microbe_stage/components/MicrobeControl.cs

```diff
@@ -195,4 +195,55 @@ public static void SetMoveSpeed(this ref MicrobeControl control, float speed)
             control.QueuedSlimeSecretionTime = 3 * random.NextSingle();
         }
     }
+
+    public static void SetMucocystState(this ref MicrobeControl control,
+        ref OrganelleContainer organelleInfo, in Entity entity, bool state)
+    {
+        if (entity.Has<MicrobeColony>())
+        {
+            ref var colony = ref entity.Get<MicrobeColony>();
+
+            // TODO: is it a good idea to allocate a delegate here?
+            colony.PerformForOtherColonyMembersThanLeader(m =>
+                m.Get<MicrobeControl>()
+                    .SetMucocystState(ref m.Get<OrganelleContainer>(), m, state));
+        }
+
+        // It might be tricky as there is now mucocyst upgrade that doesn't emit slime normally
+        if (organelleInfo.Mucocysts == null || organelleInfo.Mucocysts.Count < 1)
+            return;
+
+        // Must have at least one mucocyst
+        if (organelleInfo.Mucocysts?.Count < 1)
```

Comment:
As mucocyst state is not really needed, for performance reasons the mucosyst shouldn't be a list, but just an int similarly to how `AgentVacuoleCount` is handled.